### ISSUE TITLE
Infinite Load

### ISSUE THREAD

**Issue by rfajfar** [Ippytraxx/gnome-twitch#89]
I'm getting infinite **Loading stream**. The chat works, the video does not load.
I've installed from source on Fedora 23.

Ideas?

**Comment by Ippytraxx** [comment]
I had problems with Twitch yesterday due to the MLG Major in CS which might be what you were experiencing as well. Could you please confirm whether this problem still exists in a day or two when there is no big event running.

**Comment by rfajfar** [comment]
I've already installed **gnome-twitch** from source a few weeks ago and I was having the same problem. I thought that there was just a temporary problem with the code on **GitHub** or something. Now I've installed it again from source and its still not working.

Maybe I have a problem with `gstreamer` or something.

I'm getting this warning when opening a stream:

> Warning : "Calling Inhibit failed: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.gnome.SessionManager was not provided by any .service files"

Everything else seems to be alright.
